mod-kb-ebsco-java/src/main/resources/domain/mod-kb-ebsco-java/features/packages.feature

```diff
@@ -0,0 +1,98 @@
+Feature: Packages
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/vnd.api+json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/vnd.api+json' }
+
+  @Undefined
+  Scenario: Get all Packages
+    * print 'undefined'
+
+  @Undefined
+  Scenario: GET all Packages filtered by custom with 200 on success
```

Comment:
GET a list of custom Packages with 200 on success

Reply:
I want the scenarios for one endpoint to look the same

`GET all custom Packages with 200 on success` ?